qase-python-commons/src/qaseio/commons/testops.py

```diff
@@ -335,24 +345,32 @@ def _send_result(self, result: Result):
             if result.get_suite_title():
                 case_data["suite_title"] = "\t".join(result.get_suite_title().split("."))
 
+            result_model = ResultCreate(
+                status=result.execution.status,
+                stacktrace=result.execution.stacktrace,
+                time_ms=result.execution.duration,
+                comment=result.message,
+                attachments=[attach.hash for attach in attached],
+                defect=self.defect,
+                steps=steps,
+                param=result.params
+            )
+
+            test_ops_id = result.get_testops_id()
```

Comment:
It's called `testops_id` in many places throughout the code.

Reply:
fixed